pages/about.md

```diff
@@ -10,4 +10,4 @@ The US-RSE community blog collects articles and blog posts from personal and gro
 To join and have your blog feed part of the RSE see the [blog repo README](https://github.com/USRSE/blog) for criteria and instructions.  We welcome contributions from anyone who considers themselves to work in an RSE capacity or who publishes posts with applicable content.  See our definition of an RSE [here](https://us-rse.org/what-is-an-rse/).  
 
 Disclaimer:
-Blog feeds are curated nightly to collect new posts and archive old posts.  As a result, we cannot guarantee that anyone has reviewed any posts for content or applicability.  Opinions expressed within each post are those of the author(s) and do not necessarily reflect the opinions of the US-RSE.  Offensive or inappropriate posts should be reported to blog@us-rse.org or a pull request should be submit to remove the offending feed. 
+Blog feeds are curated nightly to collect new posts and archive old posts.  As a result, we cannot guarantee that anyone has reviewed any posts for content or applicability.  Opinions expressed within each post are those of the author(s) and do not necessarily reflect the opinions of the US-RSE.  Offensive or inappropriate posts should be reported to contact@us-rse.org or submit a pull request to remove the offending feed. 
```

Comment:
```suggestion
Blog feeds are curated nightly to collect new posts and archive old posts.  As a result, we cannot guarantee that anyone has reviewed any posts for content or applicability.  Opinions expressed within each post are those of the author(s) and do not necessarily reflect the opinions of the US-RSE.  Offensive or inappropriate posts should be reported to contact@us-rse.org if you wish to report anonymously. You can also submit a pull request to remove the offending feed if you are comfortable doing so publicly.
```

Reply:
How about this? I wrote the original and it was awkwardly stated, and I think we need to distinguish between it the person wants to remain anonymous or not.